packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -76,33 +80,43 @@ function buildSeparatorBackgroundColors(props) {
   }
 }
 
-const StyledStepper = styled.div`
+const StyledStepper = styled.div<{ isVertical?: boolean; hasLabels?: boolean }>`
   display: flex;
-  flex: 1;
-  flex-direction: column;
+  flex: ${props => (props.isVertical ? '0 0 auto' : 1)};
+  width: ${props => props.isVertical && props.hasLabels && '128px'};
```

Comment:
Lets avoid the hardcoded 128px. We expect our users to throw the Stepper instead a container of whatever size they want